plugins/woocommerce-admin/client/task-lists/fills/products/use-create-product-by-type.ts

```diff
@@ -35,23 +32,19 @@ export const useCreateProductByType = () => {
 		setIsRequesting( true );
 
 		if ( type === 'physical' ) {
-			const momentDate = moment().utc();
-			const year = momentDate.format( 'YYYY' );
-			const month = momentDate.format( 'MM' );
-			const assignment = await loadExperimentAssignment(
-				`woocommerce_product_creation_experience_${ year }${ month }_v1`
-			);
-
 			if ( isNewExperienceEnabled ) {
 				navigateTo( { url: getNewPath( {}, '/add-product', {} ) } );
 				return;
 			}
+
+			const assignment = await loadExperimentAssignment(
+				'woocommerce_product_creation_experience_202306_v2'
```

Comment:
This will limit the experiment name to just this month.  This is good if we want to limit to this one experiment after release, but if we want to close this experiment and create a new one mid-release (e.g., `woocommerce_product_creation_experience_202307_v2`) we would need to keep these months dynamic.

@pmcpinto Did we come to a decision on whether or not we want to run these experiments on a per month basis or per release?

Reply:
@joshuatf we should do it on a per release basis, to avoid limiting us to only about 2 weeks per month, considering that the release always happens in the middle of the month.

Reply:
The reason why I changed this to something fixed is because it does not matter we make the experiment name dynamic we will have to change the behavior of the variation control or treatment since them are custom per experiment so the code needs to be touched anyway. Also that approach does not scale when we want to have multiple experiment enabled at the same time.